packages/react-integration/demo-app-ts/src/components/demos/TableDemo/TableCompoundExpandableDemo.tsx

```diff
@@ -196,20 +196,23 @@ export class TableCompoundExpandableDemo extends React.Component<TableProps, Tab
 
   onExpand(event: React.MouseEvent, rowIndex: number, colIndex: number, isOpen: boolean) {
     const newRows = Array.from(this.state.rows);
-    const rowCells = newRows[rowIndex].cells;
+    const rowCells = Array.from(newRows[rowIndex].cells);
+    const thisCell = (rowCells as ICell[])[colIndex];
```

Comment:
Ah, fair enough! Thanks

Reply:
Getting back to this. I'm realizing that actually this assertion is incorrect, because the inferred type (`(React.ReactNode | IRowCell)[]`) is accurate, considering some of the cells above are plain strings. This is stuff I inherited from the previous demo code and didn't really intend to mess with too much... I may just revert these changes. Looking into it.

Reply:
Ah, but for the expandable columns, they are all `IRowCell`. Makes sense.